CONTRIBUTING.md

```diff
@@ -378,3 +378,29 @@ Test SSH keys, login form the drone node should not require typing in a password
 cd $GOPATH/src/github.com/vmware/docker-volume-vsphere/
 drone exec --trusted --yaml .drone.dev.yml -i ~/.ssh/id_rsa -e VM1=<ip VM1> -e VM2=<ip VM2> -e ESX=<ip ESX>
 ```
+
+## Capturing coverage
+Coverage is captured using make coverage target (On CI it is called using drone script).
+User can configure the setup and use this target to see the coverage.
+### Setup ESX box to install python coverage package
+* Install https://coverage.readthedocs.io/en/coverage-4.4.1/ on your machine using pip
+- scp the coverage package dir installed on ur machine i.e. <Python Location>/site-packages/coverage to /lib64/python3.5/site-packages/ on ESX box.
```

Comment:
would suggest to add the sample command instead of text .. sample command would be handy. Command reference would be easy. :)